RELEASING.md

```diff
@@ -6,18 +6,20 @@
 4. run `loghub` and update the `CHANGELOG.md` file:
 
 ```
-loghub pytroll/pyorbital -u <username> -st v0.8.0 -plg bug "Bugs fixed" -plg enhancement "Features added" -plg documentation "Documentation changes"
+loghub pytroll/pysorbital --token <personal access token (see https://github.com/settings/tokens)>  -st v<previous version> -plg bug "Bugs fixed" -plg enhancement "Features added" -plg documentation "Documentation changes"
+
 ```
 
 Don't forget to commit!
 
 5. Create a tag with the new version number, starting with a 'v', eg:
 
 ```
-git tag v0.22.45 -m "Version 0.22.45"
+git tag -a v0.22.45 -m "Version 0.22.45"
 ```
 
 See [semver.org](http://semver.org/) on how to write a version number.
 
 6. push changes to github `git push --follow-tags`
-7. Verify travis tests passed and deployed sdist and wheel to PyPI
+
+7. Verify github actions passed and code deployed to PyPI
```

Comment:
You'll need to add something about making a github release here. git tag -> verify tests pass -> github release -> verify PyPI

Reply:
Oh yes, thanks. Realize I haven't yet released after we switched from Travis to Github actions!